e2e/tests/datasets/dataset-export-py-notebook.spec.ts

```diff
@@ -45,6 +41,6 @@ describe('Export Dataset to Notebook Test', () => {
 
     // Delete notebook.
     const analysisPage = await notebookPreviewPage.goAnalysisPage();
-    await analysisPage.deleteResource(notebookName, ResourceCard.Notebook);
+    await analysisPage.deleteResourceFromTable(notebookName, ResourceCard.Notebook);
```

Comment:
Should we replace all existing instances of `deleteResource()` and remove it?

(for a future PR)

Reply:
there still are some skipped tests, will remove it in upcoming PR